src/software/simulated_tests/robot_halt.py

```diff
@@ -0,0 +1,46 @@
+import software.python_bindings as tbots
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class RobotHalt(Validation):
```

Comment:
This validation in its current form seems to be a duplicate (negation) of [robot_speed_threshold.py](https://github.com/UBC-Thunderbots/Software/blob/master/src/software/simulated_tests/robot_speed_threshold.py). #2556 tagged below is also now closed.

Reply:
bump

Reply:
file was no longer being used, it should be safe deleted now